### PR TITLE
[OSF-8228] Fix responsiveness on navbar for long names.

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose 

There are a few pixels still causing a bad two level collapse on the navbar for long names.

screen shot 2017-12-07 at 10 27 33 am

## Osf-side PR 
https://github.com/CenterForOpenScience/osf.io/pull/8013

## Changes

Adds new class with new media query that has special overflow behavior. 

## Side Effects 

None that I know of.

## Tickets

https://openscience.atlassian.net/browse/OSF-8228

### CODE REVIEW

**Comment by brianjgeiger** [comment]
@sloria Could you review?

**Comment by Johnetordoff** [comment]
Talking to @jamescdavis about this, this might not need a fix.

**Comment by Johnetordoff** [comment]
@jamescdavis Can you look at this?

File: css/osf-style.css [hunk 1]
```diff
@@ -1435,6 +1435,20 @@ button.nav-user-dropdown {
   color: #9cd59c !important;
 }
 
+.nav-profile-name {
+  max-width: 500px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (jamescdavis) [inline css/osf-style.css]: To get ellipsis to work, you need to also add `white-space: nowrap;`

File: css/osf-style.css [hunk 2]
```diff
@@ -1435,6 +1435,25 @@ button.nav-user-dropdown {
...
+@media (max-width: 1200px) {
+  .nav-profile-name {
+    max-width: 300px
+  }
+}
+
+.osf-profile-image {
+  float: left;
+}
+
```
Comment (jamescdavis) [inline css/osf-style.css]: All of these changes should be made to the .scss files in the sass/ directory and this css file should be recompiled.

**Pull request merged**